src/test/resources/s3-valid-inbound-cases.xml

```diff
@@ -66,6 +66,22 @@
 		<integration:poller fixed-rate="1000"/>
 	</int-aws:s3-inbound-channel-adapter>
 
+	<int-aws:s3-inbound-channel-adapter
+			id="validInboundWithProxy"
+			bucket="TestBucket"
+			propertiesFile="testawscredentials.properties"
+			temporary-suffix=".temp"
+			local-directory="${foo.dir:/}"
+			remote-directory="remote"
+			accept-sub-folders="true"
+			max-objects-per-batch="100"
+			file-name-regex="[A-Za-z0-9]+\\.txt"
+			channel="inbound"
+			auto-startup="false"
+            proxyHost="localhost"
+            proxyPort="80">
```

Comment:
Thanks for your time and feedback. 

If above is the case, then only modification would be to AbstractAmazonS3Operations to have a constructor to accept Client Configuration. Please let me know if this true and I can make INTEXT-194 for 1.0 Version. can you please also let me know when is the plan for GA to 1.0 version?

Thanks Again.